external/fv3fit/fv3fit/emulation/compositions/blended.py

```diff
@@ -0,0 +1,54 @@
+from typing import Callable, Mapping
+import tensorflow as tf
+
+from fv3fit.keras.adapters import ensure_dict_output
+
+TensorDict = Mapping[str, tf.Tensor]
+
+
+def blended_model(
+    model_a: tf.keras.Model,
+    model_b: tf.keras.Model,
+    mask_func: Callable[[TensorDict], tf.Tensor],
+    inputs: Mapping[str, tf.keras.Input],
+) -> tf.keras.Model:
+    """
+    Take two models and blend their outputs given a mask function.
+
+    The masking operation is broadcastable so take care to ensure
+    that large tensors aren't created by omitting a feature
+    dimension from the mask, when singleton feature fields exist.
+
+    Args:
+        model_a: model to use at points where mask == True
+        model_b: model to use at points where mask == False
+        mask_func: masking function determining model output selection.  the
+            function uses the model inputs and should return a mask tensor
```

Comment:
For clarity can you instead say "the function uses the blended model's `inputs`"? At first I read this as the `mask_func` inputs had to be in the individual models' inputs, which I think is not the case e.g. the example script adds the mask func arg `latitude` to the inputs returned by `get_model_inputs`.